src/answer.ts

```diff
@@ -48,6 +48,7 @@ const template = renderTemplate`
 <body style="max-width: 1000px; margin: 0 auto; padding: 0 1em 0 1em;">
     <header>
         <h1><a href="${"url"}">${"title"}</a></h1>
+        <img class="origin" src="${"avatar_url"}" />
```

Comment:
It seems should belong to the `<h2 rel="author">` below
While you are on it, can you adjust the layout as the following?
```
 ┌─────────┐┌──────────────┐
 │         ││              │
 │         ││              │
 │ Avatar  │└──────────────┘
 │         │┌──────────────┐
 │         ││              │
 └─────────┘└──────────────┘
```

Reply:
it require some more css to do it with flex, I will try.

Reply:
![image](https://github.com/user-attachments/assets/ae30ba0a-a0ca-4928-8f40-551f0425238c)